### PR TITLE
Press refresh

### PR DESCRIPTION
**Pull request by shamimbinnur**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This is the new UI refresh for the https://opensauced.pizza/press
Now every logo has two buttons to download in both SVG and PNG formats.

Deploy preview link: https://deploy-preview-46--opensauced-landing.netlify.app/press

## Mobile & Desktop Screenshots/Recordings

Screenshot 2022-09-26 at 6 29 51 PM
Screenshot 2022-09-26 at 6 30 02 PM

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: components/OpenSaucedLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        pngLogo: string,
+        isBlackBG: boolean,
+    }
+}
+
+const OpenSaucedLogo:FC<Props> = ({logo}) => {
+    const { title, description, pngLogo, svgLogo, isBlackBG} = logo;
+
+    return (
+    <div className='w-full tablet:max-w-[290px]'>
```
Comment (NsdHSO) [inline components/OpenSaucedLogo.tsx:17]:
```suggestion
    <div className='w-full tablet:max-w-xs'>
```

max-w-xs = 320px

File: components/OpenSaucedLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <div className={`flex h-[164px] w-full justify-center items-center ${ isBlackBG ? "bg-black" : "bg-none"} ` }>
```
Comment (NsdHSO) [inline components/OpenSaucedLogo.tsx:18]:
```suggestion
        <div className={`flex h-40 w-full justify-center items-center ${ isBlackBG ? "bg-black" : "bg-none"} ` }>
```

h-40 = 160px

File: components/OpenSaucedLogo.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <div className='h-[100px] w-auto'>
```
Comment (NsdHSO) [inline components/OpenSaucedLogo.tsx:19]:
```suggestion
            <div className='h-24 w-auto'>
```

h-24 = 96px
h-28 = 112px

File: components/OpenSaucedLogo.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                <img className='w-full h-full ' src={svgLogo} alt={title} />
```
Comment (NsdHSO) [inline components/OpenSaucedLogo.tsx:20]:
```suggestion
                <img className='w-full h-full' src={svgLogo} alt={title} />
```

File: components/OpenSaucedLogo.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            </div>
+        </div>
+
+        <div className='mt-[20px]'>
```
Comment (NsdHSO) [inline components/OpenSaucedLogo.tsx:24]:
```suggestion
        <div className='mt-5'>
```

mt-5 = 20px

**Review by NsdHSO — CHANGES_REQUESTED** [review]
Please where you use px replace with classes from tailwind. 
Please take a look here where @bdougie created this [issue](https://github.com/open-sauced/hot/issues/267)

**Comment by 0-vortex** [comment]
Good catches @NsdHSO LGTM with those changes 👍

**Review by 0-vortex — APPROVED** [review]
LGTM! 👍

File: components/PressHeading.tsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+interface Props {
+  headingData : {
+    title : string,
+    subtitle: string,
+  }
+}
+const PressHeading:FC<Props> = ({headingData}) => {
+  const {title, subtitle } = headingData;
+  return (
+    <div className='py-[4rem] tablet:py-[8rem]'>
```
Comment (bdougie) [inline components/PressHeading.tsx:12]: these rems need to be replaced with tailwinds classes. i.e. `py-2` or whatever the closest. 

There was a tweet that describes why this is important recently. https://twitter.com/t3dotgg/status/1574906239993028610

File: components/PressHeading.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+      <div>
+        <h1 className='text-white font-bold tracking-tight mb-2 text-[2rem] tablet:text-[3rem]'>{title}</h1>
```
Comment (bdougie) [inline components/PressHeading.tsx:14]: don't use rems. use classes

File: components/OpenSaucedLogo.tsx [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    <div className='w-full tablet:max-w-xs'>
+        <div className={`flex h-40 w-full justify-center items-center ${ isBlackBG ? "bg-black" : "bg-none"} ` }>
+            <div className='h-24 w-auto'>
+                <img className='w-full h-full' src={svgLogo} alt={title} />
+            </div>
+        </div>
+
+        <div className='mt-5'>
+            <h2 className='text-[20px] tracking-tight mb-[5px] text-gray-700 font-bold'>{title}</h2>
+            <p className='text-[1rem] tracking-tight text-gray-700'>{description}</p>
```
Comment (bdougie) [inline components/OpenSaucedLogo.tsx:26]: remove rems and px. use native tailwinds class

File: components/OpenSaucedLogo.tsx [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+        </div>
+
+        <div className='flex gap-x-2 pt-4'>
+            <a href={svgLogo as unknown as string} title="ImageName">
+                <button className='text-white text-[18px] font-semibold bg-gradient-to-r px-4 py-1 rounded-md from-orange-500 to-orange-400'>SVG</button>
+            </a>
+            <a href={pngLogo as unknown as string} title="ImageName">
+                <button className='text-white text-[18px] font-semibold bg-gradient-to-r px-4 py-1 rounded-md from-orange-500 to-orange-400'>PNG</button>
```
Comment (bdougie) [inline components/OpenSaucedLogo.tsx:34]: remove px

**Review by bdougie — CHANGES_REQUESTED** [review]
We need to move towards reusable tailwinds classes and away from hardcoding px and rems

**Comment by shamimbinnur** [comment]
@bdougie I am gonna try rewriting the code and replacing it with classes as much as possible to do. We might still need to use rem in some cases tho.

**Comment by bdougie** [comment]
> We might still need to use rem in some cases tho.

Need to know why and justification in the comments. Need to consider rems is the equivalent to `!mportant`. Should be used sparingly.

**Pull request merged**